sync/class.jetpack-sync-options.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/**
+ * Simple class to read/write to the options table, bypassing 
+ * problematic caching with get_option/set_option
+ **/
+
+class Jetpack_Sync_Options {
+
+	static function delete_option( $name ) {
+		global $wpdb;
+		$wpdb->query( $wpdb->prepare( "DELETE FROM $wpdb->options WHERE option_name = %s", $name ) );
+	}
+
+	static function write_option( $name, $value, $autoload = false ) {
+
+		$autoload_value = $autoload ? 'yes' : 'no';
+
```

Comment:
That seems like a good idea, but it's hard to know how to do that without _also_ doing a read, which adds an extra query to the process. Presumably, if we do a write with the same data that's already there, the database can optimise-away the write itself.